src/main/java/in/projecteka/consentmanager/consent/ConsentArtefactQueryGenerator.java

```diff
@@ -33,14 +35,14 @@
                 Tuple.of(requestId,
                         consentArtefact.getConsentId(),
                         patientId,
-                        JsonObject.mapFrom(consentArtefact),
+                        new JsonObject(from(consentArtefact)),
                         consentArtefactSignature,
                         ConsentStatus.GRANTED.toString()));
         Query insertHIPCA = new Query(INSERT_HIP_CONSENT_ARTEFACT_QUERY,
                 Tuple.of(requestId,
                         hipConsentArtefact.getConsentDetail().getConsentId(),
                         patientId,
-                        JsonObject.mapFrom(hipConsentArtefact.getConsentDetail()),
+                        new JsonObject(from(hipConsentArtefact.getConsentDetail())),
```

Comment:
since we are doing this, existing detail won't work. You need to ask someone to clean up the  database in AWS.